### PR TITLE
feat: add google api client secret json as cli option

### PR DESCRIPTION
**Pull request by kotzendekrabbe**
fix isssue #9

### CODE REVIEW

File: google-calendar-auth.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ var TOKEN_DIR = (process.env.HOME || process.env.HOMEPATH ||
 var TOKEN_PATH = TOKEN_DIR + 'calendar-nodejs-quickstart.json';
 
 var calendar = google.calendar('v3');
-var clientSecret = './googleApi_clientSecret.json';
+//var clientSecret = './googleApi_clientSecret.json';
```
Comment (MartinHelmut) [inline google-calendar-auth.js]: You should remove this comment.

File: readme.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ https://developers.google.com/google-apps/calendar/quickstart/nodejs
 ## Usage instructions ##
 
 ```node
-node ./cli.js --calID 'yourGoogleCalenderID' --meetupApiKey 'yourMeetupApiKey'
+node meetsy --calID 'yourGoogleCalenderID' --meetupApiKey 'yourMeetupApiKey' --secret './googleApi_clientSecret.json'
```
Comment (MartinHelmut) [inline readme.md]: `node meetsy` => `meetsy ...`. Also you should change the installation instruction from "clone repo" to "npm install -g meetsy".

File: google-calendar-auth.js (hunk 1 above)
Comment (kotzendekrabbe) [reply google-calendar-auth.js]: yes, you're right. thank you

File: readme.md [hunk 3]
```diff
@@ -23,13 +23,10 @@ e.g.: you can create a google calendar for your company to share Meetup events w
...
 ## Installation ##
 
-### Get meetsy ###
-You can clone or download meetsy by using the "Clone or Download" Button above.
-
 ### Install meetsy ###
 
 ```node
-npm install
+npm install -g meetsy
```
Comment (MartinHelmut) [inline readme.md]: Now you also have to change the sentence "save this data at project root and rename it to googleApi_clientSecret.json" in the bullet points, maybe something like: "save this JSON file anywhere for later use as `--secret` (remember the path to the file!)"

Comment (kotzendekrabbe) [reply readme.md]: dammit...
thank you so much

Comment (kotzendekrabbe) [reply readme.md]: ready

**Pull request merged**